README.md

```diff
@@ -14,9 +14,9 @@ npm install structured-source
 ## Usage
 
 ```js
-const  StructuredSource = require('structured-source');
+import { StructuredSource } from "structured-source"; 
```

Comment:
This is Breaking Change.

User need to use named import.

```diff
- import StructuredSource from "structured-source";
+ import { StructuredSource } from "structured-source";
```

This will improve compatibility between transpiler and native ESM.